SuperBuild/External_ITK.cmake

```diff
@@ -104,7 +104,6 @@ if(NOT DEFINED ITK_DIR AND NOT Slicer_USE_SYSTEM_${proj})
     -DITK_LEGACY_REMOVE:BOOL=OFF   #<-- Allow LEGACY ITKv4 features for now.
     -DITK_LEGACY_SILENT:BOOL=ON    #<-- Silence for initial ITKv5 migration.
```

Comment:
Probably turn this OFF sometime. Is now a good time?

Reply:
Yeah I was thinking the same thing. Was planning to do this next to see what gets shown. Same with trying `ITK_LEGACY_REMOVE` as ON.

Reply:
Maybe just silence now, and legacy when updating to 5.2.0 final? Flipping this switch should be its own commit, for ease of finding it in the log.

Reply:
Currently prepping testing with latest ITK with 5.2.0 almost released. Just been having to solve these various build issues with using Visual Studio 16.9 to build Slicer and ITK.